projects/ReOxide/services/reoxided/tests/basic.nix

```
@@ -0,0 +1,40 @@
+{
+  sources,
+  pkgs,
+  ...
+}:
+{
+  name = "reoxide";
+
+  nodes = {
+    machine =
+      { ... }:
+      {
+        imports = [
+          sources.modules.ngipkgs
+          sources.modules.programs.reoxide
+          sources.modules.services.reoxided
+          sources.examples.ReOxide."Enable reoxided"
+          (sources.inputs.nixpkgs + "/nixos/tests/common/x11.nix")
+        ];
+      };
+  };
+
+  testScript =
+    { nodes, ... }:
+    let
+      simple_plugin = "${pkgs.reoxide-plugin-simple}";
```

I feel like this is better, since it points exactly to the plugin, as opposed to the plugin's directory:

```suggestion
      simple_plugin = "${pkgs.reoxide-plugin-simple}/lib/libsimple.so";
```